### DISCUSSION TITLE
Smoother Auto-Play Animation

### DISCUSSION THREAD

**Discussion by dagim-shimelis** [naver/egjs-flicking#695] [Q&A] [answered]
I've implemented the auto-play plugin and it works perfectly but I want it to stop snapping while changing positions. 
I was wondering whether there is a way to create a smoother animation experience where the auto-play changes without the components snapping into place.

**Comment by malangfox** [comment, marked as answer]
Hello @dagim-shimelis.

You can change the movement of autoplay using the [options related to animation](https://naver.github.io/egjs-flicking/Options#animation).
In this [demo](https://codepen.io/malangfox/pen/YzeMmEg), I made an example where autoplay has different animations.

If you modify the options related to animation, you will be able to create smoother motion.
